hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/storage/BlockOutputStream.java

```diff
@@ -80,6 +82,12 @@ public class BlockOutputStream extends OutputStream {
       LoggerFactory.getLogger(BlockOutputStream.class);
   public static final String EXCEPTION_MSG =
       "Unexpected Storage Container Exception: ";
+  public static final String INCREMENTAL_CHUNK_LIST = "incremental";
+  public static final KeyValue INCREMENTAL_CHUNK_LIST_KV =
+      KeyValue.newBuilder().setKey(INCREMENTAL_CHUNK_LIST).build();
+  public static final String FULL_CHUNK = "full";
+  public static final KeyValue FULL_CHUNK_KV =
+      KeyValue.newBuilder().setKey(FULL_CHUNK).build();
```

Comment:
Can you put INCREMENTAL_CHUNK_LIST, FULL_CHUNK in some common places. I remember there is one definition too in BlockManagerImpl.

Reply:
This FULL_CHUNK_KV metadata is not welly put on chunks with full content.